### DISCUSSION TITLE
Creating a custom block to use Cards

### DISCUSSION THREAD

**Discussion by scottw12** [vanjarosoftware/Vanjaro.Platform#1278] [Q&A] [answered]
I want to have a reusable block that a editor can drag onto the page and then edit the text. At one time we could create a custom block and have it appear in the list of  predefined blocks..

I want simple code:
  <div class="card" style="width: 18rem;">
    <img src="..." class="card-img-top" alt="...">
    <div class="card-body">
      <p class="card-text">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
    </div>
  </div>

Will I be able to do this without creating an APP?  How can I add new blocks to the Vanjaro menu bar?    

I know I can use custom code to create the content but I need a editable area so text can be updated. I would like to add the text block into the custom code.

**Comment by Tychodewaard** [comment, marked as answer]
First of all: I agree a 100%. The custom code blocks were awesome :-) 

Currently, the way to go is create a section with all the Vanjaro goodies. And save that section as a custom block.

![image](https://user-images.githubusercontent.com/9091997/118851834-76c1fe00-b8d2-11eb-9762-d64f9a6b0f68.png)

**Reply by scottw12** [reply]
Ya it was nice to have the ability to edit the html albeit it had a lot of Vanjaro code in it. It still was a nice solution to create a block and even add it to the BLOCKS section rather than to have to go to a custom blocks.
